developer_manual/examples/storage-backend/OCA/MyStorageApp/Storage/MyStorage.php

```diff
@@ -0,0 +1,209 @@
+<?php
+
+namespace OCA\MyStorageApp\Storage;
+
+use \OCP\Files\Storage\StorageAdapter;
+use Icewind\Streams\IteratorDirectory;
+use Icewind\Streams\CallbackWrapper;
+
+// for this storage we use a fake library
+use \SomeVendor\FakeStorageLib;
+
+class MyStorage extends StorageAdapter
+{
```

Comment:
Adding a custom, ownCloud code style to phpStorm to help prevent me missing these in future.